save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/HistoryView.kt

```diff
@@ -86,7 +86,20 @@ class HistoryView : RComponent<HistoryProps, State>() {
                         }
                     }
                 }
-                column("date", "Date") {
+                column("startDate", "Start time") {
+                    buildElement {
+                        td {
+                            a(href = getHrefToExecution(it.value.id)) {
+                                +(it.value.startTime.let {
+                                    Instant.fromEpochSeconds(it, 0)
```

Comment:
Please extract date formatting into a function, because it's the same for two columns

Reply:
Fixed